mitiq/cdr/clifford_training_data.py

```diff
@@ -171,8 +169,7 @@ def _select(
                       method_select = 'gaussian'.
         random_state: Random state for sampling.
     """
-    if random_state is None:
-        random_state = np.random  # type: ignore
+    random_state = random_state or np.random.RandomState()
```

Comment:
Took me a while to get this shortcut :-) Nice!
Unfortunately, there is a problem if the user sets `0` as a seed. In that case `random_state` is considered as `False` and is unexpectedly replaced by a new random state.